postgresql-17.yaml

```diff
@@ -43,11 +43,16 @@ environment:
       - net-tools
       - openldap-dev
       - openssl-dev
+      - python-${{vars.py-version}}-dev
```

Comment:
please use `py${{vars.py-version}}-build-base-dev`. `python-X-dev` will land a `/usr/bin/python3` that points at `python3.13`. But things that build with it will inherent a dependency on `python3`, not `python3.13`. This means that when `python3` becomes `python3.14`, things will break.